tests/lib/basket.js

```diff
@@ -13,13 +13,22 @@ define([
 
   var LOOKUP_URL = API_URL + '/lookup-user/?email=';
 
+  var requestAttempts = 0;
+
   function waitUntilUserIsRegistered(email) {
-    console.log('Waiting for %s to register at: %s', email, API_URL);
+    if (requestAttempts > 2) {
```

Comment:
Is it possible that we'd be waiting for two different email addresses at the same time? If so, this output wouldn't be what we're looking for– we'd need to index by email or something.